aspnetcore/fundamentals/routing/samples/6.0/RoutingSample/Snippets/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace RoutingSample.Snippets
+{
+    public static class Program
+    {
+        public static void RequireHost(WebApplication app)
+        {
+            // <snippet_RequireHost>
+            app.MapGet("/", () => "Contoso").RequireHost("contoso.com");
+            app.MapGet("/", () => "AdventureWorks").RequireHost("adventure-works.com");
+
+            app.MapHealthChecks("/healthz").RequireHost("*:8080");
```

Comment:
@Rick-Anderson Who should I ask to review whether or not the three calls to to `app.MapXXX` above is a good replacement for the following 5.0 code?:

```csharp
public void Configure(IApplicationBuilder app)
{
    app.UseRouting();

    app.UseEndpoints(endpoints =>
    {
        endpoints.MapGet("/", context => context.Response.WriteAsync("Hi Contoso!"))
            .RequireHost("contoso.com");
        endpoints.MapGet("/", context => context.Response.WriteAsync("AdventureWorks!"))
            .RequireHost("adventure-works.com");
        endpoints.MapHealthChecks("/healthz").RequireHost("*:8080");
    });
}
```

From a PU perspective, I want to just make sure we want to show returning the `string` rather than using `HttpContext.Response.WriteAsync(...)`. There are a few examples in this topic that I can apply the same approach to, if that's what we want.

Reply:
@pranavkm  who's the right resource to review the .NET 6 routing changes?

Reply:
This change looks fine.